codice/ddf
# DDF-2125 Add the ability to inject default values into metacards during ingest

#### What does this PR do?
Add the ability to inject default values into metacards during ingest.
- Move the validation JSON parser into its own bundle
- Update the validation JSON parser to handle default attribute values
- Add a default attribute value API
- Updates the catalog framework to inject default values for missing attributes on creates and updates

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@rzwiefel @glenhein 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith @kcwire

#### How should this be tested?
Create a JSON file that includes some default attribute values as shown in the documentation, ingest some metacards, and verify that the default values were added where appropriate.

#### What are the relevant tickets?
[DDF-2125](https://codice.atlassian.net/browse/DDF-2125)

#### Checklist:
